src/themes/default/tokens/space.js

```diff
@@ -0,0 +1,3 @@
+const space = {}
```

Comment:
Would be genuinely curious to see how you think this should be used.

Reply:
Thinking this would/could be used for padding/margin values. I'm not 100% sure how we can safely migrate to this coming from the theme instead of using `majorScale` because today we already pass unitless numbers to `margin*` props. It would make it impossible whether to use the theme scales or treat the number as a pixel value.